ably/rest/channel.py

```diff
@@ -1,19 +1,20 @@
 from __future__ import absolute_import
 
-import calendar
 import logging
 
 import six
+import json
 from six.moves.urllib.parse import urlencode, quote
 
 from ably.http.httputils import HttpUtils
 from ably.http.paginatedresult import PaginatedResult
 from ably.types.message import Message, message_response_handler, make_encrypted_message_response_handler
-from ably.types.presence import presence_response_handler
+from ably.types.presencemessage import presence_response_handler
```

Comment:
Should this not follow the styleguide and use CapWords, https://www.python.org/dev/peps/pep-0008/#class-names